packages/dev/s2-docs/pages/index.mdx

```diff
@@ -1,30 +0,0 @@
-import {Layout} from '../src/Layout';
```

Comment:
I think we should replace this page with some info about the status of the website (that it's in beta) and links to the various libraries for now. @dannify anything specific you want it to say?

Reply:
Agreed, I'll update